sources/packages/backend/apps/db-migrations/src/sql/DisbursementValues/Add-restriction-amount-and-id-subtracted-cols.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+  sims.disbursement_values
+ADD
+  COLUMN restriction_amount_subtracted NUMERIC(8, 2);
+
+COMMENT ON COLUMN sims.disbursement_values.restriction_amount_subtracted IS 'Amount that is subtracted because of a restriction.';
+
+ALTER TABLE
+  sims.disbursement_values
+ADD
+  COLUMN restriction_id_subtracted INT REFERENCES sims.restrictions(id);
+
+COMMENT ON COLUMN sims.disbursement_values.restriction_id_subtracted IS 'Restriction id that was placed for the student due to which the award amount was reduced.';
```

Comment:
I would avoid using the  "was placed". The BCSL restriction will be the one that potentially will be placed and case a subtraction, but the subtraction can happen due to an existing restriction, for instance, "Stop BC funding".
```
Restriction id due to which the award amount was reduced.
```